holidayextras/plugin-elephant
# Web9040 fraud tools

#### What does this PR do? (please provide any background)
This stores the userAgent string into the table that links orders to baskets. We used to store this into breakDataVersions and it proved useful to debug when we were getting live issues that we couldn't recreate locally.

#### What tests does this PR have?
None - model update

#### How can this be tested?
Roberto has used the updated endpoint so you can see the userAgent being set on his booking 6VG4ZQF

#### Any tech debt?
no

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![]()


---

**Reviewer** JordanH1994

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
